src/ngraph/frontend/onnx_import/utils/common.cpp

```diff
@@ -66,6 +73,53 @@ namespace ngraph
                     std::make_shared<default_opset::ShapeOf>(value_shape),
                     default_opset::Constant::create(element::i64, {}, {step}));
             }
+
+            ONNX_NAMESPACE::TypeProto get_proto_type(element::Type type, Shape shape)
+            {
+                ONNX_NAMESPACE::TypeProto target_type;
+                switch (type)
+                {
+                case element::Type_t::f32:
+                    target_type.mutable_tensor_type()->set_elem_type(
+                        ONNX_NAMESPACE::TensorProto_DataType_FLOAT);
+                    break;
+                case element::Type_t::u8:
+                    target_type.mutable_tensor_type()->set_elem_type(
+                        ONNX_NAMESPACE::TensorProto_DataType_UINT8);
+                    break;
+                }
+
+                for (auto dim : shape)
+                {
+                    target_type.mutable_tensor_type()->mutable_shape()->add_dim()->set_dim_value(
+                        dim);
+                }
+                return target_type;
+            }
+
+            std::vector<std::shared_ptr<ngraph::Node>>
+                get_extanded_function(ONNX_NAMESPACE::NodeProto* new_node,
+                                      ONNX_NAMESPACE::GraphProto graph,
+                                      int opset_version)
+            {
+                const auto* schema = ONNX_NAMESPACE::OpSchemaRegistry::Schema(
+                    new_node->op_type(), opset_version, "");
+                const ONNX_NAMESPACE::FunctionProto* func = schema->GetFunction();
+
+                FunctionExpandHelper(*new_node, *func, graph);
+
+                graph.mutable_node()->erase(graph.node().begin());
+
+                // Save graph to file
```

Comment:
This comment seems to be obsolete, you're not saving to a file.

Reply:
Done